server/schemas/item.graphql

```diff
@@ -118,19 +118,21 @@ input ItemQuery {
 }
 
 extend type Query {
-  items(schemaId: ID!, sort: ItemSort, pagination: Pagination): ItemConnection!
+  items(modelId: ID!, pagination: Pagination): ItemConnection!
```

Comment:
where was sort gone?

Reply:
`FindByModel` method doesn't have sort in the usecase layer, I added sort to all layers.